File: C/Lista 3/1048 - Aumento de salário.c
#include <stdio.h>

int main(void)
{
    double salario, reajustePercentual, reajuste;
    scanf(" %lf", &salario);
    
    if (salario > 2000){
        reajustePercentual = 0.04;
        
    }else if (salario > 1200 && salario <= 2000){
        reajustePercentual = 0.07;
        
    }else if (salario > 800 && salario <= 1200){
        reajustePercentual = 0.1;
        
    }else if (salario > 400 && salario <= 800){
        reajustePercentual = 0.12;
        
    }else if (salario > 0 && salario <= 400){
        reajustePercentual = 0.15;
        
    }
    
    reajuste = salario * reajustePercentual;
    
    printf("Novo salario: %.2f\n", salario + reajuste);
    printf("Reajuste ganho: %.2f\n", reajuste);
    printf("Em percentual: %.0f %\n", reajustePercentual*100);
    
    return 0;
    
}

File: Python/Lista 3/1048 - Aumento de Salário.py
sal = float(input())

if 0 < sal <= 400:
    percentual = 0.15

elif 400 < sal <- 800:
    percentual = 0.12

elif 800 < sal <= 1200:
    percentual = 0.1

elif 1200 < sal <= 2000:
    percentual = 0.07

elif sal > 2000:
    percentual = 0.04

reajuste = sal * percentual
sal += reajuste

print("Novo salario: {:.2f}\nReajuste ganho: {:.2f}\nEm percentual: {:.0f} %".format(sal, reajuste, percentual*100))
